co2eq/purchase/index.test.js

```diff
@@ -128,3 +128,13 @@ test(`test household appliance for DK in DKK`, () => {
     (1150 / 7.4644) * (103.3 / 95.9) * 0.817390437852872
   );
 });
+
+test(`test diesel 10L`, () => {
```

Comment:
You could make it more clear what we're testing for here:

```suggestion
test(`test non-monetary units`, () => {
```